jupyterlab_bigquery/src/components/shared/bq_table.tsx

```diff
@@ -45,6 +37,55 @@ const localStyles = stylesheet({
   },
 });
 
+// TODO: style for dark mode. Currently the disabled buttons are not visible.
+const StyledIconButton = withStyles({
+  root: {
+    color: 'var(--jp-ui-font-color1)',
+    '&$disabled': {
+      color: 'var(--jp-ui-font-color3)',
+    },
+  },
+})(IconButton);
+
+const StyledTableCell = withStyles({
+  root: {
+    color: 'var(--jp-ui-font-color1)',
+    whiteSpace: 'nowrap',
+    textOverflow: 'ellipsis',
+    overflow: 'hidden',
+    maxWidth: '500px',
+    fontSize: '13px',
+    BASE_FONT,
+    border: 0,
+  },
+})(TableCell);
+
+// TODO: style for dark mode. Currently the page selection dropdown when highlighted is unreadable.
+export const StyledPagination: React.ComponentType<any> = withStyles({
+  root: {
+    backgroundColor: 'var(--jp-layout-color0)',
+    color: 'var(--jp-ui-font-color1)',
+    fontSize: '13px',
+    borderTop: 'var(--jp-border-width) solid var(--jp-border-color2)',
+  },
+  selectIcon: {
+    color: 'var(--jp-ui-font-color1)',
+  },
+  menuItem: {
+    color: 'var(--jp-ui-font-color1)',
+    backgroundColor: 'var(--jp-layout-color0)',
+    '&$selected': {
+      backgroundColor: 'var(--jp-layout-color2)',
+      '&:hover': {
+        backgroundColor: 'var(--jp-layout-color2)',
+      },
+    },
+    '&:hover': {
+      backgroundColor: 'var(--jp-layout-color2)',
+    },
+  },
```

Comment:
Here's where I'm having issues styling the selected `menuItem` within the `TablePagination` component.

Reply:
I took this from https://codesandbox.io/s/9t45t?file=/demo.js:478-492. Try changing this to:

```
  menuItem: {
    color: 'var(--jp-ui-font-color1)',
    backgroundColor: 'var(--jp-layout-color0)',
    '&.Mui-selected': {
      backgroundColor: 'var(--jp-layout-color2)',
      '&:hover': {
        backgroundColor: 'var(--jp-layout-color2)',
      },
    },
    '&:hover': {
      backgroundColor: 'var(--jp-layout-color2)',
    },
  },
```

![image](https://user-images.githubusercontent.com/3050838/92046900-9966b380-ed51-11ea-8c4f-269b10404439.png)

Reply:
Awesome! Thanks!